### DISCUSSION TITLE
How to attach vscode to docker container from command line

### DISCUSSION THREAD

**Discussion by marvinav** [microsoft/vscode-docker#3609] [Q&A] [answered]
Is it possible to open vscode attached to docker from command line (bash)?
I cant find any information about this in docker extension FAQ and vscode guide how to use containers.

**Comment by bwateratmsft** [comment, marked as answer]
I'll transfer this to the repo for Remote - Containers, as the "Attach VSCode" feature is provided by that extension.

EDIT: I guess I cannot transfer this since it is a discussion. Can you open an issue at https://github.com/microsoft/vscode-remote-release?
